cypress/integration/pages/articles/tests.js

```diff
@@ -35,11 +34,12 @@ export const testsThatFollowSmokeTestConfig = ({
 }) => {
   let articlesData;
   describe(`Running tests for ${service} ${pageType}`, () => {
-    before(async () => {
-      articlesData = await fetchArticlePageData(service, variant).then(
-        ({ body }) => body,
-      );
+    before(() => {
+      cy.getPageData(service, pageType, variant).then(({ body }) => {
```

Comment:
As mentioned in the comments above, can we change the signature of the call here, so that we pass in objects by name, and we also perhaps hardcode the pageType to match what the BFF is expecting?

```suggestion
      cy.getPageData({ service, pageType: 'article', variant }).then(({ body }) => {
```